src/index.ts

```diff
@@ -1,13 +1,16 @@
-import type {MessageAttributeMap, MessageAttributeValue} from 'aws-sdk/clients/sns';
+import type {MessageAttributeValue} from '@aws-sdk/client-sns';
```

Comment:
Maybe let's just copy this type to not force users install `@aws-sdk/client-sns` if they still use `aws-sdk`?

<img width="837" alt="CleanShot 2022-02-07 at 11 21 53@2x" src="https://user-images.githubusercontent.com/3817380/152769773-d14702eb-c9fb-4d58-a66b-d2b084d217be.png">

Reply:
And leave inline comment with the original type import, so we know from where to update it in the future